cmd/ethrex/l2/options.rs

```diff
@@ -301,46 +318,45 @@ impl Default for CommitterOptions {
 #[derive(Parser)]
 pub struct ProofCoordinatorOptions {
     #[arg(
-        long = "proof-coordinator-l1-private-key",
-        default_value = "0x39725efee3fb28614de3bacaffe4cc4bd8c436257e2c8bb887c4b5c4be45e76d",
+        long = "proof-coordinator.l1-private-key",
         value_name = "PRIVATE_KEY",
         value_parser = utils::parse_private_key,
         env = "ETHREX_PROOF_COORDINATOR_L1_PRIVATE_KEY",
-        help_heading = "L1 Prover Server options",
+        help_heading = "Proof coordinator options",
         long_help = "Private key of of a funded account that the sequencer will use to send verify txs to the L1. Has to be a different account than --committer-l1-private-key.",
     )]
     pub proof_coordinator_l1_private_key: SecretKey,
     #[arg(
-        long = "proof-coordinator-listen-ip",
+        long = "proof-coordinator.addr",
         default_value = "127.0.0.1",
         value_name = "IP_ADDRESS",
-        env = "ETHREX_PROOF_COORDINATOR_LISTEN_IP",
-        help_heading = "L1 Prover Server options",
+        env = "ETHREX_PROOF_COORDINATOR_LISTEN_ADDRESS",
+        help_heading = "Proof coordinator options",
         help = "Set it to 0.0.0.0 to allow connections from other machines."
     )]
     pub listen_ip: IpAddr,
     #[arg(
-        long = "proof-coordinator-listen-port",
+        long = "proof-coordinator.port",
         default_value = "3900",
         value_name = "UINT16",
         env = "ETHREX_PROOF_COORDINATOR_LISTEN_PORT",
-        help_heading = "L1 Prover Server options"
+        help_heading = "Proof coordinator options"
     )]
     pub listen_port: u16,
     #[arg(
-        long,
+        long = "proof-coordinator.send-interval-ms",
         default_value = "5000",
         value_name = "UINT64",
         env = "ETHREX_PROOF_COORDINATOR_SEND_INTERVAL_MS",
```

Comment:
I'd remove the `-ms` from the arg and env var name and add it as a help or as part of the `value_name`

Reply:
Same in other params. However, I'd keep the `_ms` suffix in the variable name so it's clear in the code

Reply:
I agree. Saw this in #2771 and liked it.

Reply:
[fa07756](https://github.com/lambdaclass/ethrex/pull/2744/commits/fa0775655727d1362c382cd5b86ead26b238d17b)

Reply:
Should we add the time unit (ms) in the `help` or `value_name`?

Reply:
Definitively.